core/src/navigation/LeftNav.html

```diff
@@ -213,9 +289,82 @@
     },
     methods: {
       handleClick(node) {
+        if (this.getIsSemiCollapsed()) {
+          this.closePopupMenu();
+        }
         this.fire('handleClick', { node });
       },
 
+      handleIconClick(nodeOrNodes, el) {
+        if (this.getIsSemiCollapsed()) {
+          event.stopPropagation();
+
+          let selectedCategory;
+          let sideBar = document.getElementsByClassName('fd-app__sidebar')[0];
+
+          if (nodeOrNodes.metaInfo && nodeOrNodes.metaInfo.label) {
```

Comment:
Due to the babel plugin, we can use now `(nodeOrNodes.metaInfo?.label)`.

Reply:
done

Reply:
I had to revert it because integration tests were failing with this change